client/ayon_core/plugins/publish/collect_resources_path.py

```diff
@@ -77,16 +80,23 @@ def process(self, instance):
 
         # This is for cases of Deprecated anatomy without `folder`
         # TODO remove when all clients have solved this issue
-        template_data.update({
-            "frame": "FRAME_TEMP",
-            "representation": "TEMP"
-        })
+        template_data.update({"frame": "FRAME_TEMP", "representation": "TEMP"})
 
-        publish_templates = anatomy.get_template_item(
-            "publish", "default", "directory"
+        template_name = get_publish_template_name(
+            project_name=template_data["project"]["name"],
+            host_name=get_current_host_name(),
```

Comment:
Isn't this just `instance.context.data["hostName"]` preferably?

Reply:
Ah could be. It is so hard to collect all of this in consistent way. Sometimes it is much easier (although inefficient) just to get it again than looking for collector or dumping data in pyblish. I was actually expecting it along other things in Anatomy data (but it is not there). Traits for publishing instances I say!